x/perpetual/keeper/repay.go

```diff
@@ -43,6 +43,10 @@ func (k Keeper) Repay(ctx sdk.Context, mtp *types.MTP, pool *types.Pool, ammPool
 	// This is for accounting purposes, mtp.Custody gets reduced by borrowInterestPaymentCustody and funding fee. so msg.Amount is greater than mtp.Custody here. So if it's negative it should be closed
 	if mtp.Custody.IsZero() || mtp.Custody.IsNegative() {
 		k.DestroyMTP(ctx, *mtp)
+		err = k.hooks.AfterPositionDestroyed(ctx, mtp.GetAccountAddress(), mtp.AmmPoolId, mtp.Id)
```

Comment:
there is already AfterPerpetualPositionClosed, we can send closing Ratio as extra parameter in it and use that?

Reply:
done